client/tx/search.go

```diff
@@ -102,17 +102,44 @@ func SearchTxRequestHandlerFn(ctx context.CoreContext, cdc *wire.Codec) http.Han
 		tag := r.FormValue("tag")
 		if tag == "" {
 			w.WriteHeader(400)
-			w.Write([]byte("You need to provide a tag to search for."))
+			w.Write([]byte("You need to provide a tag or an address to search for."))
 			return
 		}
+		keyValue := strings.Split(tag, "=")
+		key := keyValue[0]
+		value := keyValue[1]
+		if strings.HasSuffix(key, "_bech32") {
+			bech32address := strings.Trim(value, "'")
+			prefix := strings.Split(bech32address, "1")[0]
+			bz, err := sdk.GetFromBech32(bech32address, prefix)
+			if err != nil {
+				w.WriteHeader(400)
+				w.Write([]byte(err.Error()))
+				return
+			}
+
+			tag = strings.TrimRight(key, "_bech32") + "='" + sdk.Address(bz).String() + "'"
+		}
 
-		tags := []string{tag}
-		output, err := searchTx(ctx, cdc, tags)
+		txs, err := searchTxs(ctx, cdc, []string{tag})
```

Comment:
Do we have reason to support multiple tags (joined with `AND` in the `searchTxs` function above), or is that unnecessary?